### ISSUE TITLE
Are OME-Zarr tests in scope

### ISSUE THREAD

**Issue by jluethi** [haesleinhuepf/human-eval-bia#98]
@haesleinhuepf given your call for contributing tests: Are additional small tests on extracting information from OME-Zarrs in scope for this repo? I'd be happy to add some to encourage LLMs to learn about NGFF ;)

I saw you test the very basic opening of Zarr files here: https://github.com/haesleinhuepf/human-eval-bia/blob/main/test_cases/open_zarr.ipynb

If similar tests for reading labels from an OME-Zarr, reading a tiny image and e.g. summing up its intensities etc. are of interest, I can come up with a tiny example OME-Zarr that can fit in the repo and working on some notebook tests.

**Comment by haesleinhuepf** [comment]
Absolutely! Please send a pull-request for such test cases! Consider writing small test cases testing one single functionality - and add a complex workflow involving multiple steps. 

Great examples using ngff and zarr are very welcome, also because I couldn't write them 😃
